src/controllers/profile.js

```diff
@@ -0,0 +1,189 @@
+import profileImage from '../services/imageUpload';
+import UserService from '../services/user.service';
+import successResponse from '../utils/successResponse';
+import ErrorResponse from '../utils/errorResponse';
+import { verifyToken } from '../helpers/token';
+
+const { findUser, updateUser } = UserService;
+
+class Profile {
+  static async getProfile(req, res, next) {
+    try {
+      const {
+        firstName,
+        lastName,
+        email,
+        userName,
+        isVerified,
+        facebookId,
+        googleId,
+        age,
+        gender,
+        dob,
+        nationality,
+        department,
+        preferredLanguage,
+        preferredCurrency,
+        address,
+        lineManager,
+        phoneNumber,
+        maritalStatus,
+        isComplete,
+      } = req.user;
+
+      return successResponse(res, 200, `Successfully retrieved profile`, {
+        firstName,
+        lastName,
+        email,
+        userName,
+        isVerified,
+        facebookId,
+        googleId,
+        age,
+        gender,
+        dob,
+        nationality,
+        department,
+        preferredLanguage,
+        preferredCurrency,
+        address,
+        lineManager,
+        phoneNumber,
+        maritalStatus,
+        isComplete,
+      });
+    } catch (error) {
+      return ErrorResponse.internalServerError(
+        res,
+        `Error occured while fetching your profile ${error.message}`
+      );
+    }
+  }
+
+  static async completeProfile(req, res, next) {
+    try {
+      const updated = await updateUser(req.user, {
+        ...req.body,
+        profileImage: '',
```

Comment:
How is this profileImage linked with user? I can see you did not add it in the model or migration